internal/modifier/gated.go

```diff
@@ -46,31 +46,31 @@ func NewFeatureGatedModifier(logger logger.Interface, cfg *config.Config, image
 	driverRoot := cfg.NVIDIAContainerCLIConfig.Root
 	devRoot := cfg.NVIDIAContainerCLIConfig.Root
 
-	if cfg.Features.IsEnabled(config.FeatureGDS, image) {
+	if cfg.Features.IsEnabledInEnvironment(config.FeatureGDS, image) {
```

Comment:
Question -- Is `config.FeatureGDS` the environment variable,`NVIDIA_GDS` ? Or is it the feature name `gds`?

Reply:
On a quick pass, this appears to be `gds` when I would expect us to be passing `NVIDIA_GDS`.

Reply:
Okay I am wrong. We are passing the feature name, and `IsEnabledInEnvironment` will translate this to the envvar name.

Reply:
the feature name is `gds`, but this calls:
```
	case FeatureGDS:
		return fs.GDS.isEnabledWithEnvvarOverride("NVIDIA_GDS", in...)
```
Where the envvar is used.